hiveos/h-manifest.conf

```diff
@@ -0,0 +1,26 @@
+####################################################################################
+###
+### kaspa-miner
+### https://github.com/tmrlvi/kaspa-miner/releases
+###
+### Hive integration: Merlin
+###
+####################################################################################
+
+# The name of the miner
+CUSTOM_NAME=kaspa-miner
+
+# Optional version of your custom miner package
+CUSTOM_VERSION=0.1.3
+CUSTOM_BUILD=0
+CUSTOM_MINERBIN=kaspa-miner-v0.1.3-GPU-with-workload-linux-amd64
```

Comment:
Is there a way to get `latest`? Maybe I can change configuration to facilitate it